guides/common/modules/ref_migrating-from-foreman_webhooks.adoc

```diff
@@ -0,0 +1,12 @@
+[id="migrating-webhooks_{context}"]
+= Migration to Foreman Webhooks
+
+The legacy `foreman_hooks` provides full access to model objects that the webhooks plugin does not intentionally provide.
+
+The scope of what is available is limited by the safemode and all objects and macros are both subject to API stability promise and are fully documented.
+
+The number of events triggered by webhooks is substantially less than with `foreman_hooks`.
+
+The webhooks plugin is designed to be safe, hooks are processed asynchronously so there is minimal risk of tampering with internals of the system. It is not possible to migrate from `foreman_hooks` without creating payloads for each individual webhook script.
+
+The webhooks plugin, however, comes with several example payload templates, and together with shellhooks plugin migration should be straightforward. Both script and payload templates must be customized to achieve similar results.
```

Comment:
This sentence doesn't entirely make sense, especially around this bit
```
together with shellhooks plugin migration should be straightforward. Both script and payload templates must be customized to achieve similar results.
```

I thought the migration that we are talking about here is from Shellhooks to webhooks. 
Please implement the one-sentence-per line rule also.